### PR TITLE
Revised the namespace alias qualifier article

### PR DESCRIPTION
**Pull request by pkulikov**
Contributes to #10741

### CODE REVIEW

File: docs/csharp/language-reference/operators/namespace-alias-qualifier.md [hunk 1]
```diff
@@ -1,45 +1,64 @@
...
+  {
+      public static forwpf::Point Convert(forwinforms::Point point) => new forwpf::Point(point.X, point.Y);
+  }
+  ```
 
-The `::` operator can also be used with a *using alias directive*:
+- An [extern alias](../keywords/extern-alias.md).
+- The `global` alias, which is the global namespace alias. The global namespace is the namespace that contains namespaces and types that are not declared inside a named namespace. When used with the `::` qualifier, the `global` alias always references the global namespace, even if there is the user-defined `global` namespace alias.
+  
+  The following example uses the `global` alias to access the .NET <xref:System> namespace, which is a member of the global namespace. Without the `global` alias, the user-defined `System` namespace, which is a member of the `MyCompany.MyProduct` namespace, would be accessed:
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/namespace-alias-qualifier.md]: With that example in this article, I suggest to remove and redirect the following programming guide article:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/how-to-use-the-global-namespace-alias

Comment (BillWagner) [reply docs/csharp/language-reference/operators/namespace-alias-qualifier.md]: I agree. Do you want to add it to this PR, or make a separate PR?

**Review by BillWagner — COMMENTED** [review]
This LGTM @pkulikov 

Do you want to add the removal of the programming guide article to this PR, or make a separate PR? If you prefer a separate PR, this is ready to :shipit:

File: docs/csharp/language-reference/operators/namespace-alias-qualifier.md (hunk 1 above)
Comment (pkulikov) [reply docs/csharp/language-reference/operators/namespace-alias-qualifier.md]: @BillWagner I prefer to make it as a separate PR. I'll resolve the merge conflict with TOC and then it's ready to ship.

**Review by BillWagner — APPROVED** [review]
Thanks again @pkulikov 

Once the build finishes, I'll :shipit:

**Comment by pkulikov** [comment]
@BillWagner thanks for review; I've resolved the merge conflict. Now, it should be ready.

**Pull request merged**